demonstrations/tutorial_ml_classical_shadows.py

```diff
@@ -0,0 +1,871 @@
+r"""
+Learning ground state of 2D antiferromagnetic Heisenberg model
+==============================================================
+
+.. meta::
+    :property="og:description": Learning ground state of 2-D
+    antiferromagnetic Heisenberg model.
+    :property="og:image": https://pennylane.ai/qml/_images/ml_classical_shadow.png
+
+.. related::
+
+   tutorial_classical_shadows Classical Shadows
+
+
+*Author: Utkarsh. Posted: XX. Last Updated: XX March 2022*
+
+
+In general, simulating an :math:`n`-qubit quantum mechanical system
+becomes classically intractable as the number of classical bits required
+to store the state of the system scale exponentially in :math:`n`. This
+inhibits us from using classical simulation of larger quantum systems to
+estimate their properties. In the past, this challenge has been
+addressed by the quantum community using classical shadow formalism,
+which allows one to build a concise classical description of the state
+of a quantum system. Recently, it has been shown by Hsin-Yuan Huang et
+al. [[#preskill]_] that combining classical shadows with classical
+machine learning enables using ML methods to efficiently predict
+properties of the quantum systems such as the expectation value of
+Hamiltonian, correlations functions, entanglement entropies, etc.
```

Comment:
```suggestion
Hamiltonian, correlations functions, and entanglement entropies.
```
Since you're saying "such as" it's ok to give a few concrete examples. In general it's problematic to use "etc." since it makes the text sound vague. You'll hardly ever see it in scientific writing